lambda_configs/bless_deploy.cfg

```diff
@@ -0,0 +1,65 @@
+# This section and its options are optional
+[Bless Options]
+# Number of seconds +/- the issued time for the certificate to be valid
+certificate_validity_after_seconds = 120
+certificate_validity_before_seconds = 86400
+# Minimum number of bits in the system entropy pool before requiring an additional seeding step
+entropy_minimum_bits = 2048
+# Number of bytes of random to fetch from KMS to seed /dev/urandom
+random_seed_bytes = 256
+# Set the logging level
+logging_level = INFO
+# Comma separated list of the SSH Certificate extensions to include. Not specifying this uses the ssh-keygen defaults:
+# certificate_extensions = permit-X11-forwarding,permit-agent-forwarding,permit-port-forwarding,permit-pty,permit-user-rc
+# Username validation options are described in bless_request_user.py:USERNAME_VALIDATION_OPTIONS
+# Configure how bastion_user names are validated.
+# username_validation = useradd
+# Configure how remote_usernames names are validated.
+# remote_usernames_validation = principal
+# Configure a regex of blacklisted remote_usernames that will be rejected for any value of remote_usernames_validation.
+# remote_usernames_blacklist = root|admin.*
+# Number of seconds +/- the issued time for the server certificates to be valid
+# server_certificate_validity_before_seconds = 120
+# server_certificate_validity_after_seconds = 31536000
+# Configure how server certificate hostnames are validated
+# hostname_validation = url
+
+# These values are all required to be modified for deployment
+[Bless CA]
+# You must set an encrypted private key password for each AWS Region you deploy into
+# for each aws region specify a config option like '{}_password'.format(aws_region)
+us-east-1_password = AQICAHibDsNZpMt+ZjdpBHrsbZd/fbjVqiF/CSncj9Y/Xvea3AH0LX6HgJQYQQsPtqkp96OvAAAAZjBkBgkqhkiG9w0BBwagVzBVAgEAMFAGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMfK7NV2NIPX7FzAtQAgEQgCNZPf6d5MKXW+tIon2DMIOtLpqtQHY+TCIGUwo6Dp0DOE+3Iw==
```

Comment:
i believe there is a way around it i.e. putting this in env var or so; not important but just so that you know